library/Booster/Definition/Attributes/Reader.hs

```diff
@@ -188,7 +193,7 @@ instance HasAttributes ParsedSymbol where
             <*> hasConcreteEvaluators
             <*> pure Nothing
             <*> smt
-            <*> (attributes .:? "hook")
+            <*> hook
```

Comment:
... or maybe just `(encodeUtf8 <$> attributes  .:? "hook")` ? :golfing_man:

Reply:
It actually needs to be `(fmap encodeUtf8 <$> attributes  .:? "hook")`. I though that was a little too much!

Reply:
I've settled on
```
            hook = do
                hooked <- attributes .:? "hook"
                pure (encodeUtf8 <$> hooked)
```